client/settings/deposits/index.js

```diff
@@ -114,13 +114,15 @@ const CustomizeDepositSchedule = () => {
 					value={ depositScheduleInterval }
 					onChange={ handleIntervalChange }
 					options={ depositIntervalsOptions }
+					__nextHasNoMarginBottom
 				/>
 				{ depositScheduleInterval === 'monthly' && (
 					<SelectControl
 						label={ __( 'Date', 'woocommerce-payments' ) }
 						value={ depositScheduleMonthlyAnchor }
 						onChange={ setDepositScheduleMonthlyAnchor }
 						options={ monthlyAnchors }
+						__nextHasNoMarginBottom
```

Comment:
`36px default size for wp.components.SelectControl is deprecated since version 6.8 and will be removed in version 7.1.`
```suggestion
						__nextHasNoMarginBottom
						__next40pxDefaultSize
```